runelite-client/src/main/java/net/runelite/client/plugins/virtuallevels/VirtualLevelsPlugin.java

```diff
@@ -45,13 +47,23 @@
 public class VirtualLevelsPlugin extends Plugin
 {
 	private static final String TOTAL_LEVEL_TEXT_PREFIX = "Total level:<br>";
+	private static final String VIRT_TOTAL_LEVEL_TEXT_PREFIX = "Virtual total level:<br>";
```

Comment:
Is there enough room in the total level box to show this text? Can you show a screenshot of this?

Reply:
https://gyazo.com/01e01bbeeb7662ef85cc723d7012e69e = current version,
https://gyazo.com/f349a93a279f34cffb4e5d778da91549 = another one i tried.
Both of them clip out a little bit, im not sure what would fit better though.
Maybe just leave show total level anyway? Not sure..

Reply:
Yeah, it seems like it would be better to leave the text be.